tower_cli/resources/job_template.py

```diff
@@ -99,17 +99,12 @@ class Resource(models.SurveyResource):
         help_text='On write commands, perform extra POST to the '
                   'survey_spec endpoint.')
 
-    @resources.command
-    def create(self, fail_on_found=False, force_on_exists=False,
-               extra_vars=None, **kwargs):
-        """Create a job template."""
+    def write(self, *args, **kwargs):
```

Comment:
Here, `create` is being converted into `write`. The user of the former was what caused the bug in the first place. The standard options were getting applied, but somehow were not functioning correctly. Over-riding the write method is extremely reliable by comparison since it is strictly used internally (not as a command), so its behavior will be predictable as the standard python behavior even if each level is using stuff from its super-class.